lib/types/orders.ts

```diff
@@ -91,6 +91,24 @@ export type Pair = {
   quoteCurrency: string;
 };
 
+export type SwapDeal = {
```

Comment:
We already define this in `Swaps.ts`, why define it again?

Reply:
I was actually thinking of leaving this in `Swaps.ts`, although in #597 I'm moving it to its own types file just for swap-related types. So if that gets merged it would be good to use that commit here and drop this type definition.